doc/examples/reconst_cti.py

```diff
@@ -0,0 +1,152 @@
+
+.. math::
+	\[K_{iso} = 3 \cdot \frac{V({\overline{D}^c})}{\overline{D}^2}\]
+        
+        where: \(K_{iso}\) is the isotropic kurtosis,
+            \(V({\overline{D}^c})\) represents the variance of the diffusion tensor raised to the power c, 
+            \(\overline{D}\) is the mean of the diffusion tensor.
+            
+The anisotropic source is calculated as: 
+
+:math:: 
+            
+            \[K_{aniso} = \frac{6}{5} \cdot \frac{\langle V_{\lambda}(D_c) \rangle}{\overline{D}^2}\]
+            
+        where: \(K_{aniso}\) is the anisotropic kurtosis, 
+            \(\langle V_{\lambda}(D_c) \rangle\) represents the mean of the variance of eigenvalues of the diffusion tensor,
+            \(\overline{D}\) is the mean of the diffusion tensor.
+            
+The microscopic source is calculated as: 
+
+:math:: 
+                \[\Psi = \frac{2}{5} \cdot \frac{D_{11}^2 + D_{22}^2 + D_{33}^2 + 2D_{12}^2 + 2D_{13}^2 + 2D_{23}^2{\overline{D}^2} - \frac{6}{5} \]
+                \[{\overline{W}} = \frac{1}{5} \cdot (W_{1111} + W_{2222} + W_{3333} + 2W_{1122} + 2W_{1133} + 2W_{2233})\]
+            
+            where \(\Psi\) is a variable representing a part of the total excess kurtosis,
+            \(D_{ij}\) are elements of the diffusion tensor,
+            \(\overline{D}\) is the mean of the diffusion tensor.
+            \{\overline{W}} is the mean kurtosis,
+            \(W_{ijkl}\) are elements of the kurtosis tensor.
+            
+and 
+
+:math:: 
+	K_{\text{intra}} = K_{\text{T}} - K_{\text{aniso}} - K_{\text{iso}}
+	
+	where K_{\text{intra}} refers to intra-compartmental kurtosis, 
+	K_{\text{T}} refers to total kurtosis, 
+	K_{\text{aniso}} refers to anisotropic source of kurtosis, 
+	K_{\text{iso}} refers to isotropic source of kurtosis
+
+In the following example we show how to fit the correlation tensor model on a real life dataset and how to estimate correlation tensor based statistics. 
+
+First, we'll import all relevant modules. 
+""" 
+import matplotlib.pyplot as plt
+import numpy as np
+import math
+
+from dipy.reconst.tests.test_qti import _anisotropic_DTD, _isotropic_DTD
+from dipy.core.sphere import Sphere, HemiSphere
+from dipy.reconst.tests.test_qti import _anisotropic_DTD, _isotropic_DTD
+import dipy.reconst.qti as qti
+from dipy.reconst.qti import (from_3x3_to_6x1)
+from dipy.reconst.dti import (decompose_tensor, mean_diffusivity)
+import dipy.reconst.cti as cti
+from dipy.io import read_bvals_bvecs
+from dipy.core.gradients import gradient_table
+from dipy.io.image import load_nifti
+"""
+CTI analysis requires specific gradient information from the data, often acquired from different gradient tables. In this example, we manually load b-values and b-vectors from two separate sets of files, 'bvals1.bval' and 'bvec1.bvec', and 'temp_data/bvals2.bval' and 'temp_data/bvec2.bvec'. 
```

Comment:
Write instead. CTI analysis requires data acquired with double diffusion encoding data acquired with different pairs of b-values and gradient directions between the two diffusion epochs according to the considerations by Henriques et al., Magn Reson Med 2021

Reply:
Add a reference to Henriques et al., Magn Reson Med 2021.

Reply:
Add also the following sentence. To run CTI we need to have separate b-value and b-vector files for each DDE diffusion epoch. In this tutorial, a sample DDE dataset and respective b-value and b-vector files for the two epochs are fetched. If you want to process your own DDE data compatible with CTI, you will have to change the above lines of code to add the paths of your data.